packages/opentelemetry-resource-detector-aws/src/detectors/AwsEc2Detector.ts

```diff
@@ -42,7 +44,7 @@ class AwsEc2Detector implements Detector {
    *
    * @param config The resource detection config with a required logger
    */
-  async detect(config: ResourceDetectionConfigWithLogger): Promise<Resource> {
+  async detect(config: any): Promise<Resource> {
```

Comment:
Does this need to be an `any`? What is the config used for?

Reply:
fixed